Sources/MapboxDirections/RouteLegAnnotation.swift

```diff
@@ -0,0 +1,49 @@
+
+import Foundation
+import CoreLocation
+
+struct RouteLegAnnotation {
```

Comment:
Note that this library avoids the term “annotation” in public API, since that term is so overloaded on Apple platforms. This struct isn’t public, but the suggestion above is to eventually make it public, in which case we’d rename it to something like RouteLegAttributes and probably nest it within RouteLeg.